tests/Keboola/Utils/SanitizeNameTest.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Keboola\Utils;
+
+class SanitizeNameTest extends \PHPUnit_Framework_TestCase
+{
+
+    public function testSanitizeName()
```

Comment:
dataprovider pls :)

Reply:
also test some value with utf8 characters (👍 or some Vietnamese, which could be real use case)